master/internal/command/command_manager.go

```diff
@@ -42,6 +44,7 @@ func (cs *commandService) GetCommands(req *apiv1.GetCommandsRequest) (*apiv1.Get
 	return resp, nil
 }
 
+// GetCommand returns the command matching the commandID.
```

Comment:
```suggestion
// GetCommand looks up a command by ID returns a summary of the its state and configuration.
```
these comments could be a little more descriptive.